x-pack/plugins/maps/common/convert_to_geojson.js

```diff
@@ -5,12 +5,12 @@
  */
 
 import _ from 'lodash';
-import { RENDER_AS } from '../../../../common/constants';
+import { RENDER_AS, GEOTILE_GRID_AGG_NAME, GEOCENTROID_AGG_NAME } from './constants';
```

Comment:
Common is getting a bit unorganized. Would you mind moving convert_to_geojson, elasticsarch_geo_utils, and es_agg_utils and elasticsearch utils folder? Include an index.ts folder so modules do not have to import into this folder.